src/questions.json

```diff
@@ -0,0 +1,321 @@
+    },
+
+       {
+        "id": 25,
+        "question": "But I've learned a lesson that stressin' and obsessin' 'bout somebody else is no fun and snakes and stones never broke my bones",
+        "option1": "Paper Rings",
+        "option2": "You Need To Calm Down",
+        "option3": "Cruel Summer",
+        "answer": "You Need To Calm Down",
+        "lyricsVideo": "https://www.youtube.com/embed/Dkk9gvTmCXY"
+    },
+
+    {
+        "id": 26,
+        "question": "I'm so sick of running as fast as I can wondering if I'd get there quicker if I was a man",
+        "option1": "The Man",
+        "option2": "The Archer",
+        "option3": "Lover",
+        "answer": "The Man",
+        "lyricsVideo": "https://www.youtube.com/embed/AqAJLh9wuZ0"
+    },
+
+    {
+        "id": 27,
+        "question": "And there's a dazzling haze, a mysterious way about you, dear. Have I known you twenty seconds or twenty years?",
+        "option1": "The Man",
+        "option2": "The Archer",
+        "option3": "Lover",
+        "answer": "Lover",
+        "lyricsVideo": "https://www.youtube.com/embed/-BjZmE2gtdo"
+    },
+
+     {
+        "id": 28,
+        "question": "But I can see us lost in the memory, August slipped away into a moment in time 'cause it was never mine",
+        "option1": "August",
+        "option2": "Seven",
+        "option3": "Style",
+        "answer": "August",
+        "lyricsVideo": "https://www.youtube.com/embed/nn_0zPAfyo8"
+    },
+
+     {
+        "id": 29,
+        "question": "Does a scorpion sting when fighting back? They strike to kill, and you know I will, you know I will",
+        "option1": "This Is Me Trying",
+        "option2": "Epiphany",
+        "option3": "Mad Woman",
+        "answer": "Mad Woman",
+        "lyricsVideo": "https://www.youtube.com/embed/6DP4q_1EgQQ"
+    },
+
+     {
+        "id": 30,
+        "question": "Does a scorpion sting when fighting back? They strike to kill, and you know I will, you know I will",
+        "option1": "This Is Me Trying",
+        "option2": "Epiphany",
+        "option3": "Mad Woman",
+        "answer": "Mad Woman",
+        "lyricsVideo": "https://www.youtube.com/embed/6DP4q_1EgQQ"
+    },
+
+    {
+        "id": 31,
+        "question": "Oh, I can't stop you putting roots in my dreamland, my house of stone, your ivy grows and now I'm covered in you",
+        "option1": "Ivy",
+        "option2": "Coney Island",
+        "option3": "'Tis The Damn Season",
+        "answer": "Ivy",
+        "lyricsVideo": "https://www.youtube.com/embed/9nIOx-ezlzA"
+    },
+
+    {
+        "id": 32,
+        "question": "And I've cleaned enough houses to know how to cover up a scene. Good thing Este's sister's gonna swear she was with me",
+        "option1": "No Body, No Crime",
+        "option2": "Delicate",
+        "option3": "'Evermore",
+        "answer": "No Body, No Crime",
+        "lyricsVideo": "https://www.youtube.com/embed/IEPomqor2A8"
+    }
```

Comment:
Awesome job!! I love that you embeded a video for each question, that's really cool!! You did a really great job on everything!!